### ISSUE TITLE
Pre-work Checklist: Developer: Ryan Gehris

### ISSUE THREAD

**Issue by RyanGehris** [hackforla/website#3913]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by RyanGehris** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

Onboarding meeting: 3 hours
Docker: 0.5 hours
Pre-work reading: 1.5 hours

My issue took me about 2 hours because I had a question regarding the prompt, I then had an issue with my github authorization token expiring, and I had to read about how to submit a PR

**Comment by RyanGehris** [comment]
Progress: I have completed every issue except for a large issue

**Comment by RyanGehris** [comment]
Progress: I just picked up another medium issue. I plan to pick up a large issue next week.

**Comment by ExperimentsInHonesty** [comment]
@Ryan, please do not work on a medium issue.  Its much harder to make the issues below Large, so each person can only have one.  I am removing your assignment from the medium issue #2288 so that you can take on a large issue

[Large backend](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22+++label%3A%22complexity%3A+large%22#column-7198257)

**Comment by jdingeman** [comment]
Hey @RyanGehris, thank you for picking up a large issue! Be sure to provide us with an update when you get the chance. Thank you!

**Comment by RyanGehris** [comment]
I am currently working on a large issue. I planned to have it completed by 4/11, but am behind after getting a take home assignment last week. I will still be trying to complete this project by 4/11.

**Comment by RyanGehris** [comment]
I was not able to complete the large issue within a two week time frame. I needed to complete a take home assignment, and now have to prepare for an interview and will be out of town immediately following that. I unassigned myself and moved the issue back to New Issue Approval column where it can be reprioritized. I also notified a Justin of my coming absence. I will be able to pick up a large issue and get back to work on 4/24/2023.

**Comment by RyanGehris** [comment]
I was recently interviewing at a company and received an offer. This week I have been onboarding and training. I plan to be more active and pick up a large issue for HackForLA starting Tuesday 5/02.

**Comment by roslynwythe** [comment]
> I was recently interviewing at a company and received an offer. This week I have been onboarding and training. I plan to be more active and pick up a large issue for HackForLA starting Tuesday 5/02.

Hi @RyanGehris that is great news about the offer!  We've been very impressed by your contributions and your PR reviews and I wish you luck at the new job!  This pre-work issue has been flagged for review by the dev leads because it has been inactive (without comments) for at least two weeks.  So please leave a comment and let us know if you will be continuing with HfLA, and if you do want to continue, please leave updates at leave every two weeks.  Thanks!

**Comment by RyanGehris** [comment]
Yes, I would like to continue contributing.

**Comment by RyanGehris** [comment]
@roslynwythe I don't think I will be able to continue volunteering for HackForLA at this time. I seem to have too many other things to juggle right now. Thank you for the kind words earlier. Is there anything I need to do other than let you or someone else know?

**Comment by roslynwythe** [comment]
Hi @RyanGehris thank you for your update and for your contributions to Hack for LA.  I will be closing this issue, but you are welcome to return at any time simply by reopening this issue, leaving a comment, removing the `2 weeks inactive` label, and moving it to the "In Progress" column.  Best of luck to you.

**Comment by HackforLABot** [comment]
## Activity Log: RyanGehris
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- RyanGehris opened: https://github.com/hackforla/website/issues/3913 at 01/31/2023, 08:58 PM PST
- RyanGehris assigned: https://github.com/hackforla/website/issues/3913 at 01/31/2023, 08:58 PM PST
- RyanGehris assigned: https://github.com/hackforla/website/issues/3895 at 02/02/2023, 12:24 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/issues/3895#issuecomment-1414335076 at 02/02/2023, 12:30 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/issues/3895#issuecomment-1414542647 at 02/02/2023, 04:19 PM PST
- RyanGehris opened: https://github.com/hackforla/website/pull/3927 at 02/02/2023, 05:14 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/issues/3913#issuecomment-1414584986 at 02/02/2023, 05:22 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/3927#issuecomment-1420176104 at 02/06/2023, 08:15 PM PST
- RyanGehris assigned: https://github.com/hackforla/website/issues/3856 at 02/09/2023, 05:57 PM PST
- RyanGehris unassigned: https://github.com/hackforla/website/issues/3856#issuecomment-1425070509 at 02/09/2023, 05:59 PM PST
- RyanGehris issue closed by PR 3927: https://github.com/hackforla/website/issues/3895#event-8485586240 at 02/09/2023, 06:54 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/3962#issuecomment-1426910791 at 02/11/2023, 04:51 PM PST
- RyanGehris submitted review: https://github.com/hackforla/website/pull/3962#pullrequestreview-1294645220 at 02/11/2023, 06:12 PM PST
- RyanGehris assigned: https://github.com/hackforla/website/issues/3694#issuecomment-1410597638 at 02/13/2023, 08:40 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/issues/3694#issuecomment-1429105870 at 02/13/2023, 08:42 PM PST
- RyanGehris unassigned: https://github.com/hackforla/website/issues/3694#issuecomment-1429105870 at 02/13/2023, 09:36 PM PST
- RyanGehris assigned: https://github.com/hackforla/website/issues/3694#issuecomment-1429105870 at 02/13/2023, 09:36 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/issues/3694#issuecomment-1429150085 at 02/13/2023, 09:38 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/issues/3694#issuecomment-1429156003 at 02/13/2023, 09:48 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/issues/3694#issuecomment-1430565588 at 02/14/2023, 04:15 PM PST
- RyanGehris opened: https://github.com/hackforla/website/pull/3977 at 02/14/2023, 04:27 PM PST
- RyanGehris PR was merged: https://github.com/hackforla/website/pull/3977#event-8528297780 at 02/15/2023, 09:03 AM PST
- RyanGehris issue closed by PR 3977: https://github.com/hackforla/website/issues/3694#event-8528298147 at 02/15/2023, 09:03 AM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/3982#issuecomment-1432249581 at 02/15/2023, 03:58 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/3980#issuecomment-1432250445 at 02/15/2023, 03:58 PM PST
- RyanGehris submitted review: https://github.com/hackforla/website/pull/3982#pullrequestreview-1300568535 at 02/15/2023, 04:15 PM PST
- RyanGehris submitted review: https://github.com/hackforla/website/pull/3980#pullrequestreview-1300578653 at 02/15/2023, 04:30 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/3962#issuecomment-1432290004 at 02/15/2023, 04:35 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/4020#issuecomment-1439414693 at 02/21/2023, 08:15 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/4020#issuecomment-1439434677 at 02/21/2023, 08:48 PM PST
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4020#pullrequestreview-1308522996 at 02/21/2023, 08:51 PM PST
- RyanGehris submitted review: https://github.com/hackforla/website/pull/3962#pullrequestreview-1310372942 at 02/22/2023, 02:27 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/4020#issuecomment-1440937929 at 02/22/2023, 02:45 PM PST
- RyanGehris assigned: https://github.com/hackforla/website/issues/3960#issuecomment-1427262247 at 02/23/2023, 10:32 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/issues/3960#issuecomment-1442869482 at 02/23/2023, 10:33 PM PST
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4034#pullrequestreview-1314350541 at 02/24/2023, 02:50 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/issues/3960#issuecomment-1449132437 at 02/28/2023, 04:30 PM PST
- RyanGehris opened: https://github.com/hackforla/website/pull/4076 at 02/28/2023, 09:55 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/4053#issuecomment-1450572158 at 03/01/2023, 09:40 AM PST
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4053#pullrequestreview-1320359604 at 03/01/2023, 09:59 AM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/4076#issuecomment-1452620496 at 03/02/2023, 02:12 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/issues/3913#issuecomment-1453911810 at 03/03/2023, 10:12 AM PST
- RyanGehris PR was merged: https://github.com/hackforla/website/pull/4076#event-8667392041 at 03/04/2023, 10:07 PM PST
- RyanGehris issue closed by PR 4076: https://github.com/hackforla/website/issues/3960#event-8667392052 at 03/04/2023, 10:07 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/4106#issuecomment-1461076588 at 03/08/2023, 04:25 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/4108#issuecomment-1461078290 at 03/08/2023, 04:27 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/4126#issuecomment-1461080052 at 03/08/2023, 04:29 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/4131#issuecomment-1461080716 at 03/08/2023, 04:30 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/4133#issuecomment-1461081236 at 03/08/2023, 04:31 PM PST
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4106#pullrequestreview-1331831688 at 03/08/2023, 04:52 PM PST
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4126#pullrequestreview-1331879774 at 03/08/2023, 06:08 PM PST
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4133#pullrequestreview-1331885187 at 03/08/2023, 06:16 PM PST
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4131#pullrequestreview-1331906694 at 03/08/2023, 06:50 PM PST
- RyanGehris commented on: https://github.com/hackforla/website/pull/4108#issuecomment-1461281145 at 03/08/2023, 09:00 PM PST
- RyanGehris opened: https://github.com/hackforla/website/pull/4182 at 03/14/2023, 02:04 PM PDT
- RyanGehris closed: https://github.com/hackforla/website/pull/4182#event-8748508028 at 03/14/2023, 02:04 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4108#pullrequestreview-1340311328 at 03/14/2023, 02:32 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4194#pullrequestreview-1345157411 at 03/16/2023, 06:48 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/pull/4228#issuecomment-1478759727 at 03/21/2023, 05:26 PM PDT
- RyanGehris assigned: https://github.com/hackforla/website/issues/2288#issuecomment-1476672283 at 03/21/2023, 05:31 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/issues/2288#issuecomment-1478764850 at 03/21/2023, 05:35 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/issues/3913#issuecomment-1478822194 at 03/21/2023, 07:06 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4228#pullrequestreview-1351675090 at 03/21/2023, 08:42 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/issues/2288#issuecomment-1483559227 at 03/24/2023, 04:07 PM PDT
- RyanGehris unassigned: https://github.com/hackforla/website/issues/2288#issuecomment-1483559227 at 03/26/2023, 05:36 PM PDT
- RyanGehris assigned: https://github.com/hackforla/website/issues/3732#issuecomment-1449413433 at 03/28/2023, 02:30 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/issues/3732#issuecomment-1487622064 at 03/28/2023, 02:32 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4304#pullrequestreview-1361954762 at 03/28/2023, 02:48 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4305#pullrequestreview-1362077374 at 03/28/2023, 05:01 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/pull/4305#issuecomment-1487787352 at 03/28/2023, 05:43 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/pull/4297#issuecomment-1487854119 at 03/28/2023, 07:20 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4297#pullrequestreview-1364165656 at 03/29/2023, 05:33 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/pull/4297#issuecomment-1489526844 at 03/29/2023, 05:44 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/pull/4340#issuecomment-1489666953 at 03/29/2023, 09:17 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/pull/4341#issuecomment-1489667515 at 03/29/2023, 09:18 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4340#pullrequestreview-1364308376 at 03/29/2023, 09:31 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4341#pullrequestreview-1364318395 at 03/29/2023, 09:44 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/pull/4297#issuecomment-1491088247 at 03/30/2023, 04:21 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4391#pullrequestreview-1373889832 at 04/05/2023, 05:14 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/pull/4391#issuecomment-1498323396 at 04/05/2023, 05:19 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4421#pullrequestreview-1373900076 at 04/05/2023, 05:25 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4425#pullrequestreview-1373907697 at 04/05/2023, 05:38 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/issues/3913#issuecomment-1500993032 at 04/08/2023, 03:57 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4444#pullrequestreview-1376858919 at 04/08/2023, 10:16 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/issues/3913#issuecomment-1503841954 at 04/11/2023, 10:51 AM PDT
- RyanGehris commented on: https://github.com/hackforla/website/issues/3732#issuecomment-1503846178 at 04/11/2023, 10:54 AM PDT
- RyanGehris unassigned: https://github.com/hackforla/website/issues/3732#issuecomment-1503846178 at 04/11/2023, 10:54 AM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4461#pullrequestreview-1379866766 at 04/11/2023, 11:36 AM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4461#pullrequestreview-1380245597 at 04/11/2023, 05:15 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4495#pullrequestreview-1382629097 at 04/12/2023, 09:56 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4495#pullrequestreview-1384423884 at 04/13/2023, 05:19 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4495#pullrequestreview-1384509213 at 04/13/2023, 07:21 PM PDT
- RyanGehris submitted review: https://github.com/hackforla/website/pull/4495#pullrequestreview-1394737618 at 04/20/2023, 01:45 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/pull/4495#issuecomment-1516923984 at 04/20/2023, 01:46 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/pull/4495#issuecomment-1517233684 at 04/20/2023, 09:14 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/issues/3913#issuecomment-1526843178 at 04/27/2023, 06:13 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/issues/3913#issuecomment-1553999422 at 05/18/2023, 10:06 PM PDT
- RyanGehris commented on: https://github.com/hackforla/website/issues/3913#issuecomment-1610572811 at 06/27/2023, 07:39 PM PDT
- RyanGehris issue closed as not planned: https://github.com/hackforla/website/issues/3913#event-9711812743 at 07/03/2023, 10:13 AM PDT
